httptools/src/main/java/org/pesmypetcare/httptools/HttpClient.java

```diff
@@ -0,0 +1,165 @@
+package org.pesmypetcare.httptools;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.google.gson.Gson;
+
+import org.pesmypetcare.httptools.exceptions.MyPetCareException;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+/**
+ * @author Santiago Del Rey
+ */
+public class HttpClient {
+    private static final String TAG = "HttpClient";
+    private static final int OK = 200;
+    private static final int CREATED = 201;
+    private static final int NO_CONTENT = 204;
+
+    /**
+     * Default constructor.
+     */
+    public HttpClient() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5538337)